examples/skr/TROUBLESHOOTING.md

```diff
@@ -53,3 +53,57 @@ err: AKV post request failed: HTTP GET failed: Post "https://<mhsm-name>.managed
 Ensure that:
 
 - the name of the mHSM is correct in the ARM template and matches the name of the mHSM in the importkeyconfig.json file
+
+## "CreateContainerRequest is blocked by policy" error (Confidential Containers AKS Only)
+
+If you are running the secure key release container on confidential containers AKS and the container pod does not run. Occasionally this could be caused by incorrect workload identity setup that results in empty environment variables that the container depends on. To debug, issue the following command: 
+
+```bash
+# This command describes the current status of the container pod
+kubectl describe pod skr 
+```
+If the error is the following: 
+
+Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create containerd task: failed to create shim task: "CreateContainerRequest is blocked by policy": unknown
+
+Issue the following commands to check the workload identity is properly enabled in the namespace where skr container pod is run: 
+
+```bash
+# Obtain the client id of the managed identity. 
+# USER_ASSIGNED_IDENTITY_NAME is the name of the managed identity used for accessing key vault and RESOURCE_GROUP is the resource group name where the managed identity resides 
+export USER_ASSIGNED_CLIENT_ID="$(az identity show --resource-group "${RESOURCE_GROUP}" --name "${USER_ASSIGNED_IDENTITY_NAME}" --query 'clientId' -otsv)" 
+# Obtain the service account detail
+kubectl get sa <service-account-name> -n <skr-container-namespace> -o yaml 
+```
+
+Check the service account `azure.workload.identity/client-id` annotation value matches the obtained `USER_ASSIGNED_CLIENT_ID` value. If they match, check the following: 
+
+```bash 
+# Obtain the AKS_OIDC_ISSUER. Replace CLUSTER_NAME and RESOURCE_GROUP with the name of the cluster skr is run and the resource group the cluster resides 
+export AKS_OIDC_ISSUER="$(az aks show -n "${CLUSTER_NAME}" -g "${RESOURCE_GROUP}" --query "oidcIssuerProfile.issuerUrl" -otsv)"
+# Obtain the federated identity detail 
+# USER_ASSIGNED_IDENTITY_NAME is the name of the managed identity used for accessing key vault
```

Comment:
Since you already have these as comments in the command above, I wouldn't repeat them.